pkg/controller/injector/k8s_binding_injector.go

```diff
@@ -60,7 +60,6 @@ func CreateK8sBindingInjector() (BindingInjector, error) {
 
 func (b *k8sBindingInjector) Inject(binding *servicecatalog.Binding, cred *brokerapi.Credential) error {
 	is := makeInjectionSet(binding, cred)
-
 	if err := b.injectSecret(is.secret); err != nil {
```

Comment:
@arschles if you don't want to do that reorg in this PR, while in here anyway, that's ok - I can follow-up with another PR if you want.